symfony/src/Controller/DebugToolsController.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Controller;
+
+use App\Form\DebugParticipantLookupType;
+use App\Service\DebugToolsService;
+use App\Service\EnvironmentService;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/s/admin/debug")
+ */
+class DebugToolsController extends AbstractController
+{
+    /**
+     * @Route("/participants", name="admin_debug_participants")
+     */
+    public function participantsAction(Request $request, EnvironmentService $env, DebugToolsService $debugToolsService)
+    {
+        if ($env->isProd()) {
+            throw $this->createNotFoundException('Page Not Found!');
```

Comment:
If we don't have a need to customize the 404 exception, you can just leave the constructor empty. (Same in the other calls below).